src/libsystemd-network/dhcp-server-internal.h

```diff
@@ -68,6 +68,9 @@ struct sd_dhcp_server {
         DHCPLease invalid_lease;
 
         uint32_t max_lease_time, default_lease_time;
+
+        sd_dhcp_server_callback_t cb;
+        void *cb_data;
```

Comment:
I'd prefer `callback` and `callback_userdata`. Those abbreviations just make this harder to read (and don't match the surrounding style). "userdata" is obvious at first sight, "data" could be many other things...

Reply:
ack

Reply:
Yeah, I think so. We should also change the type to 64 bit, so there's enough space.